spring-cloud-gcp-pubsub-stream-binder/src/test/java/com/google/cloud/spring/stream/binder/pubsub/provisioning/PubSubChannelProvisionerTests.java

```diff
@@ -266,20 +227,6 @@ void testAfterUnbindConsumer_nonAnonymous() {
     verify(this.pubSubAdminMock, never()).deleteSubscription(result.getName());
   }
 
-  @Test
-  void testProvisionConsumerDestination_concurrentTopicCreation() {
```

Comment:
Why is this no longer a valid scenario?

Reply:
because `provisionConsumerDestination` now does not have topic creation functionality